charts/internal/gvisor/values.yaml

```diff
@@ -1,4 +1 @@
-config:
-  kubernetesVersion: 1.16.1
```

Comment:
https://github.com/gardener/gardener-extension-runtime-gvisor/blob/00c4cced57b8facf4935a66a6dc832c46838b5c4/pkg/controller/controller_test.go#L48

has to be updated to 1.17+.